omnipaxos/src/storage.rs

```diff
@@ -693,19 +693,22 @@ where
         }
     }
 
+    fn create_decided_snapshot(&mut self) -> StorageResult<T::Snapshot> {
+        self.create_snapshot(self.get_decided_idx())
```

Comment:
This is incorrect because we don't take into account if a stopsign is included in the decided entries. It so we need to snapshot 1 less item.